src/e3/testsuite/fragment.py

```diff
@@ -215,6 +242,10 @@ def __init__(
         self.result_queue = []
         super().__init__(uid, driver, callback_name, slot, env)
 
+    def clear_driver_data(self) -> None:
```

Comment:
ThreadTestFragment.cllear_driver_data clears `self.data`, but ProcessTestFragment does not. As far as I can tell, neither directly manipulate `self.data`, so I'm wondering why one needs to clear it while the other one doesn't'.

Reply:
`self.data` comes from the `e3.job.Job` derivation, which only `ThreadTestFragment` inherits :-) Thanks for the review!